AppCenterDataStorage/AppCenterDataStorage/MSDataStore.h

```diff
@@ -35,11 +35,19 @@ static NSString *const MSDataStoreUserDocumentsPartition = @"user";
 static NSString *const MSDataStoreAppDocumentsPartition = @"readonly";
 
 /**
- * Time to live constants
+ * No expiration on cache.
  */
 static int const MSDataStoreTimeToLiveInfinite = -1;
+
+/**
+ * Do not cache.
+ */
 static int const MSDataStoreTimeToLiveNoCache = 0;
-static int const MSDataStoreTimeToLiveDefault = 60 * 60;
+
+/**
+ * Default expiration on cache.
+ */
+static int const MSDataStoreTimeToLiveDefault = 60 * 60 * 24;
```

Comment:
Any reason why it was changed from 1hr to 24hr?

Reply:
@jwallra Changed in PR #1503 and got merged into this one.